### PR TITLE
[script.copacetic.helper 1.0.16]

### PR DESCRIPTION
**Pull request by realcopacetic**
### Description
**1.0.16**
- Added tvchannels/radiochannels to background_slideshow()
- Error handling for clearlogos that aren't in a supported PIL mode https://github.com/realcopacetic/script.copacetic.helper/issues/3
- Added landscape to movieartwhitelist/tvshowartwhitelist recommended settings
- SlideshowMonitor class extended to accept custom paths to folders of images e.g. special://profile/backgrounds/ - it can now populate background slideshows sourced from library paths, playlists, plugin sources (e.g. themoviedbhelper) and folders
- Rebuilt wideget_move() to account for new widget settings

**1.0.15**
- Global search action to open keyboard and return value to relevant skin string.
- Minor error catching in play_all()

**1.0.14**
- Test fix for issue: IndexError: list index out of range https://github.com/realcopacetic/script.copacetic.helper/issues/5

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by realcopacetic** [comment]
I can't see anything in this PR that would prompt the add-on checker failure. I noticed @ronie pushed a PR yesterday (https://github.com/xbmc/repo-scripts/pull/2618#issuecomment-2106319754) with the same failure, so I'm not sure if this is an issue with the checker?

File: script.copacetic.helper/script.py [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
 from resources.lib.script.actions import *
-from resources.lib.utilities import clear_cache, sys
+from resources.lib.utilities import clear_cache, sys, urllib
 
 
 class Main:
     def __init__(self, *args):
         try:
             self.params = dict(arg.split('=', 1) for arg in args)
+            log(f'FUCK_IN: {self.params}', force=True)
```
Comment (basrieter) [inline script.copacetic.helper/script.py:11]: Could we rename this? We don't want little devs to read foul language 😄

**Review by basrieter — CHANGES_REQUESTED** [review]

File: script.copacetic.helper/script.py (hunk 1 above)
Comment (realcopacetic) [reply script.copacetic.helper/script.py:11]: Ahh sorry haha, thought I had removed that, as you can probably tell I was getting quite frustrated with getting something to work - will remove it asap! Thanks for checking @basrieter

Comment (realcopacetic) [reply script.copacetic.helper/script.py]: OK, updated and squashed - thanks @basrieter

**Review by basrieter — APPROVED** [review]

**Pull request merged**